src/Api/include/Api/Orbit.h

```diff
@@ -40,6 +40,10 @@
 // deploy and dynamically load liborbit.so into the target process. Orbit will then write the proper
 // function addresses into the "g_orbit_api_v<N>" table.
 //
+// NOTE: To enable manual instrumentation, please make sure that:
+//       1. The "Enable Orbit Api in target" checkbox is ticked in the capture options.
```

Comment:
Only thought I have is that `capture options` could be `"Capture Options" dialog` to be even more precise. Just a thought.

Reply:
good point, done, thanks.